documentation/anonymization.markdown

```diff
@@ -1,79 +1,75 @@
 ---
-title: "PuppetDB 3.2 » Data Anonymization"
+title: "PuppetDB 3.2: Anonymizing data"
```

Comment:
This is an established industry term we are swapping out here: https://en.wikipedia.org/wiki/Data_anonymization, is there a good reason for this?